### PR TITLE
successive KeyOps

### PR DESCRIPTION
**Pull request by adamrocker**
In this PR, merge them to one KeyOp if there are some successive KeyOps.

ex) Render the keyboard layout successful if there are some successive KeyOps.
スクリーンショット 2021-03-12 11 48 40

### CODE REVIEW

File: src/models/KeyboardModel.ts [hunk 1]
```diff
@@ -222,9 +222,15 @@ export default class KeyboardModel {
...
         } else {
-          const op = item as KeyOp;
-          const label = keyRow[++col] as string; // next item should be string(row,col)
+          let op = item as KeyOp;
+          let nextOp = keyRow[++col];
+          while (typeof nextOp != 'string') {
+            // in case of continuing KeyOp
+            op = Object.assign({}, op, nextOp);
+            nextOp = keyRow[++col];
+          }
```
Comment (yoichiro) [inline src/models/KeyboardModel.ts:231]: @adamrocker Don't we need to concern that the loop is not finished because the type of `nextOp` never be string?

**Review by yoichiro — CHANGES_REQUESTED** [review]
@adamrocker I posted a question. Could you confirm that? Thanks!

File: src/models/KeyboardModel.ts (hunk 1 above)
Comment (adamrocker) [reply src/models/KeyboardModel.ts:231]: ah..., it's definitely meaningless KEL but might happen. I'm gonna fix it. Thanks!

File: src/models/KeyboardModel.ts [hunk 2]
```diff
@@ -190,10 +190,12 @@ export default class KeyboardModel {
     let bottom = 0;
     let top = Infinity;
     keymaps.forEach((model) => {
-      right = Math.max(right, model.endRight);
-      left = Math.min(left, model.startLeft);
-      bottom = Math.max(bottom, model.endBottom);
-      top = Math.min(top, model.top);
+      if (!model.isDecal) {
```
Comment (adamrocker) [inline src/models/KeyboardModel.ts:193]: Considered the decal(hidden) key

File: src/models/KeyboardModel.ts [hunk 3]
```diff
@@ -50,19 +50,19 @@ class Current {
 
   setOp(op: KeyOp) {
     if (op.rx || op.ry) {
-      this.rx = op.rx || this.rx;
```
Comment (adamrocker) [inline src/models/KeyboardModel.ts:53]: If op.rx is 0, this.rx will be assigned

File: src/models/KeyboardModel.ts (hunk 1 above)
Comment (adamrocker) [reply src/models/KeyboardModel.ts]: and this https://github.com/remap-keys/remap/pull/393/commits/2827966ff43ca81002197135ccbb8dae3b4a802e

**Review by yoichiro — APPROVED** [review]
@adamrocker LGTM. Thanks!

**Pull request merged**